services/src/app.js

```diff
@@ -28,6 +28,9 @@ const sequelizeSwagger = require('./sequelize-swagger');
 const settings = configuration();
 const app = express(feathers());
 
+// const SUCCESS = 'OK';
+const FAILURE = 'ERROR';
```

Comment:
AFAICT, that is adding the response designed in the spec, but not checking the test actually receives this?

I tested locally using your fix, and adding the following inside the `.catch(err => {` block seems to do:

```javascript
assert.ok(err.error);
assert.ok(err.error.status);
assert.equal(err.error.status, 'ERROR');
```

Reply:
Checking this actually will be my next task